src/chrome/content/rules/bdstatic.com.xml

```diff
@@ -1,20 +1,17 @@
 <!--
 	For other Baidu coverage, see Baidu.xml.
 
-	Nonfunctional hosts in *bdstatic.com:
+	405 Not Allowed:
+		ns0.bdstatic.com	http://ns0.bdstatic.com/static/fisp_static/common/lib/mod_b818356.js
+		ns1.bdstatic.com	http://ns1.bdstatic.com/static/fisp_static/focustop/focustop/focustop_a273719.css
```

Comment:
Both urls WFM

Reply:
I have updated this comment.
Same with you now?

Reply:
Nope, I get 200 on http and https

Reply:
Test by Tor, Ok.
Directly, 405.
It's one China only issue.